api/events/swagger-api_events.yaml

```diff
@@ -66,14 +66,9 @@ paths:
                 type: object
                 properties:
                   events:
-                    type: object
```

Comment:
Why did you change this?

Reply:
There were two levels with a `count` field

Reply:
My bad